sambacc/commands/satellite/__main__.py

```diff
@@ -0,0 +1,61 @@
+#
```

Comment:
Why did we name it _\_\_main\_\_.py_ and not _main.py_? We seem to use _main.py_ elsewhere.

Reply:
Interesting..